bundles/org.openhab.binding.touchwand/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="bridge-type:touchwand:bridge">
+		<parameter name="username" type="text">
+			<label>Username</label>
+			<description>Username for TouchWand Wanderfull Hub</description>
+		</parameter>
+		<parameter name="password" type="text">
+			<context>password</context>
+			<label>Password</label>
+			<description>Password for TouchWand Wanderfull Hub </description>
+			<required>true</required>
```

Comment:
The `required` tag is deprecated. Please use the `required="true"` attribute in the `parameter` tag. Same for below.